### ISSUE TITLE
Review SchedulerData name

### ISSUE THREAD

**Issue by jskeet** [googleapis/google-cloudevents#18]
Currently the event data proto message for Scheduler events is "SchedulerData".

The event itself is `google.cloud.scheduler.job.v1.executed`.

I wonder whether it should be `SchedulerJobData`, `JobData` or `ExecutionData`. I can imagine other Scheduler-based events that may need different kinds of data.

cc @yolocs @grant 

Note: this doesn't affect the *content* of the message, just its name.

**Comment by yolocs** [comment]
That makes sense.

**Comment by grant** [comment]
`SchedulerJobExecutedData`?

**Comment by jskeet** [comment]
I'd prefer not to include the "executed" part as that wouldn't then be reusable for other things (e.g. if we had "schedule changed" or "execution failed" events). (So I don't like my own original idea of `ExecutionData` either.)

**Comment by yolocs** [comment]
So maybe something more general just like `JobData`?

**Comment by jskeet** [comment]
Yes, possibly. (Or SchedulerJobData if we want to give a bit more context.)

**Comment by yolocs** [comment]
Doesn't the proto package already scope the job to be a scheduler job?

**Comment by jskeet** [comment]
Yes, it does - it just may avoid cross-namespace type collisions. (Similarly we have PubsubMessage instead of Message - although it's just MessagePublishedData....) I don't mind *much* either way, but would be interested in others' opinions.

**Comment by yolocs** [comment]
Good point.
